sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/SystemEventMappingNames.java

```diff
@@ -467,17 +539,17 @@
             "Microsoft.Media.LiveEventIncomingStreamsOutOfSync";
 
     /**
-     * Incoming video stream out of synch event data. Schema of the data property of an EventGridEvent for a
+     * Incoming video stream out of sync event data. Schema of the data property of an EventGridEvent for a
      * Microsoft.Media.LiveEventIncomingVideoStreamsOutOfSync event.
      */
     public static final String MEDIA_LIVE_EVENT_INCOMING_VIDEO_STREAMS_OUT_OF_SYNC =
             "Microsoft.Media.LiveEventIncomingVideoStreamsOutOfSync";
 
     /**
-     * Ingest fragment dropped event data. Schema of the data property of an EventGridEvent for a
-     * Microsoft.Media.LiveEventIngestHeartbeat event.
+     * Ingest heartbeat event data. Schema of the data property of an EventGridEvent for a
+     * Microsoft.Media.LiveEventIngestHeartbeatEventData event.
      */
-    public static final String MEDIA_LIVE_EVENT_INGEST_HEARTBEAT = "Microsoft.Media.LiveEventIngestHeartbeat";
+    public static final String MEDIA_LIVE_EVENT_INGEST_HEARTBEAT = "Microsoft.Media.LiveEventIngestHeartbeatEventData";
```

Comment:
breaking changes. Do we need to keep EventData suffix consistency? 
Another example, which is the new class added.

MEDIA_LIVE_EVENT_CHANNEL_ARCHIVE_HEARTBEAT, MediaLiveEventChannelArchiveHeartbeatEventData.class

Reply:
This was fixed in a later commit - see other comment.

Reply:
Thanks. With the latest SHA commit, it is good.